### ISSUE TITLE
GREAT app but could use one important feature

### ISSUE THREAD

**Issue by agunslinger** [tabulapdf/tabula#645]
If this could be altered to not just do one selection/table in one document that would be awesome.  I need to batch extract 2 specific locations on 120 PDF forms. I can import them into Tabula well, it analyzes them perfectly, but each file imported is extracted into a separate excel sheet.  If I could combine the results of all imported and analyzed files into one spreadsheet that would be a game changer.  I would pay for that ;^)

**Comment by jeremybmerrill** [comment]
You can use [tabula-java](https://github.com/tabulapdf/tabula-java) to script the extraction from all of those PDFs into a single CSV. The Tabula GUI will generate a script for tabula-java (the `Script` download option) that you can call on each PDF, so your end result will end up looking something like this 

````
sh tabula-agunslingerspdf.sh -a something,or,other 1.pdf > shared_output.csv
sh tabula-agunslingerspdf.sh -a something,or,other 2.pdf > shared_output.csv
sh tabula-agunslingerspdf.sh -a something,or,other 3.pdf > shared_output.csv
````

**Comment by agunslinger** [comment]
Closer.  Feel free to let me know if Im too basic to cover this.  I got the tablua-java script.  It generated an sh file showing the selection coordinates for both areas I need applied to all these templated PDFs.  I am not sure what to do from here.  PResume execute tabula-0.9.0-jar-with-dependencies.jar from a command prompt?

tabula  -a 222.233,179.775,254.363,449.82 -p 1 "$1"
tabula  -a 404.303,276.93,426.488,342.72 -p 1 "$1"

**Comment by jeremybmerrill** [comment]
I made a mistake in the example I gave above, my bad, sorry about that. You'll want to download the latest tabula-java ([tabula-0.9.2-jar-with-dependencies.jar](https://github.com/tabulapdf/tabula-java/releases/download/0.9.2/tabula-0.9.2-jar-with-dependencies.jar)) and rename it to `tabula.jar`. That'll let the downloaded script work. The script contents look like what you pasted into your comment (`tabula -a 222.233,179.775,254.363,449.82 -p 1 "$1"`), and you call it from the command line with the now-fixed version in my above comment.

For the first PDF, you'd do something like this:

`sh tabula-whatever.sh 1.pdf > any_name_you_want_for_the_output_spreadsheet.csv`

and for the second,

`sh tabula-whatever.sh second.pdf > any_name_you_want_for_the_output_spreadsheet.csv`
